### DISCUSSION TITLE
New feed: Can't see people starring my repositories

### DISCUSSION THREAD

**Discussion by domfarolino** [community/community#66172] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Maybe same as https://github.com/orgs/community/discussions/66165 but worth reiterating, since the new feed is basically unusable! I won't use it until this gets fixed, unfortunately.

**Comment by AndyObtiva** [comment]
GitHub claimed they consolidated the previous feeds. However, the new feed is missing information from the old feeds (all people who star the user's projects), so saying it's consolidation is a lie. GitHub owes its users an apology for lying to them about the new feed in addition to fixing their mistake. Lying is not a good look for GitHub. Is this the Microsoft image everyone was worried would take shape after they've taken over GitHub? Prove me wrong GitHub by humbly apologizing for your mistake and fixing the new feed to support all the old feed features completely via options that users could disable/enable any way they wish.
